src/patterns/dosdp-patterns/cellPartOfAnatomicalEntity.yaml

```diff
@@ -2,25 +2,30 @@ pattern_name: cellPartOfAnatomicalEntity
 pattern_iri: http://purl.obolibrary.org/obo/cl/cellPartOfAnatomicalEntity
 
 description: A cell that is part of an anatomical entity.
-#examples: CL_0009001 compound eye retinal cell, CL_2000004 pituitary gland cell (654 total)
+# examples:
```

Comment:
This should guide usage of the template.  Please change to:

"Use this template to define a class of cells by a combination of gross type and location.  e.g. Peyer's patch B cell.|